pallets/dao/src/lib.rs

```diff
@@ -577,14 +602,19 @@ pub mod pallet {
 			// Find member and remove from Vector
 			ensure!( members.iter().any(|a| *a == *account), Error::<T>::NotMember);
 			members.retain(|a| *a != *account);
+
 			// Update Organization Members
 			<Members<T>>::insert(org_id, members);
 
 			// Find current organizations and remove org_id from MemberOf user
 			let mut current_organizations = <Pallet<T>>::member_of(&account);
 			ensure!(current_organizations.iter().any(|a| *a == org_id), Error::<T>::InvalidOrganization);
-			current_organizations.retain(|a| *a != org_id);
+
 			// Update MemberOf
+			current_organizations = current_organizations.into_iter().filter(|a| *a !=
```

Comment:
isnit this equivalent to `current_organizations.retain(|a| *a != org_id);` which was removed?

Reply:
After some reading, retain should be used if you dont mind consuming the variable. My benchmarking didnt have it any quicker but it doesnt copy the vec like filter does. :). Lazy from me.